src/components/core/wizard/wizard.tsx

```diff
@@ -0,0 +1,401 @@
+import CloseIconDark from '@assets/svg/modal/ic-close-dark.svg';
+import CloseIconLight from '@assets/svg/modal/ic-close-light.svg';
+import BackArrowIconLight from '@assets/svg/wizard/ic-back-arrow-light.svg';
+import BackArrowIconDark from '@assets/svg/wizard/ic-back-arrow-dark.svg';
+import Button from '@core/button/button';
+import Text from '@core/text/text';
+import React from 'react';
+import { styled } from 'Styles/stitches.config';
+import StepNavigation from './step-navigation';
+
+const DarkBackgroundContainer = styled('div', {
+    position: 'absolute',
+    inset: '0',
+    background: 'rgba(0, 0, 0, 0.72)',
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+
+    variants: {
+        visible: {
+            false: {
+                display: 'unset',
+                background: 'unset',
+            },
+        },
+    },
+});
+
+const WizardContainer = styled('div', {
+    position: 'relative',
+    width: '1040px',
+    height: '640px',
+    backgroundColor: '$white',
+    borderRadius: '16px',
+    display: 'flex',
+    overflow: 'hidden',
+    boxShadow: '0px 0px 24px rgba(0, 0, 0, 0.08), 0px 24px 24px rgba(0, 0, 0, 0.08)',
+
+    variants: {
+        dark: {
+            true: {
+                backgroundColor: '#0E0E0E',
+            },
+        },
+    },
+});
+
+const LeftPanel = styled('div', {
+    width: '256px',
+    height: '640px',
+    backgroundColor: '#F2F3F4',
+    boxSizing: 'border-box',
```

Comment:
Let's set this at the root level in our global styles by having a css reset plus this.

Reply:
@sean-binary I've added `boxSizingForAllChildren` to `stitches.config` and used it in WizardContainer. Please let me know if it's not what you meant 🙂